tidb-cloud/tidb-cloud-sso-authentication.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Single sign-on Authentication
+summary: Learn how to sign in via your Google workspace account and your Github Accounts in TiDB Cloud Web console.
+---
+# SSO(single sign-on) Authentication
+
+This document describes how individual or corporate users can quickly log in to TiDB CLoud Web Console via Googol workspace and GitHub accounts. Since your identity is centrally hosted on the third-party Google and GitHub platforms, TiDB cloud does not support you to modify your account password and enable MFA after logging in TiDB Cloud via SSO authentication.If you want to know about logging in through user and password , please see [Password Authentication](/tidb-cloud/tidb-cloud-password-authentication.md)
```

Comment:
```suggestion
This document describes how to quickly log in to TiDB Cloud Console via Googol workspace and GitHub accounts. 

If you log into TiDB Cloud via SSO authentication, because your identity is centrally hosted on the third-party Google and GitHub platforms, you will not be able to modify your account password and enable MFA in the TiDB console. If you want to log into TiDB Cloud through username and password, see [Password Authentication](/tidb-cloud/tidb-cloud-password-authentication.md)
```